src/Rules/StarterWeb/AI/IndividualAuth/Startup.cs

```diff
@@ -34,7 +34,7 @@ public Startup(IHostingEnvironment env)
             if (env.IsDevelopment())
             {
                 // For more details on using the user secret store see http://go.microsoft.com/fwlink/?LinkID=532709
-                builder.AddUserSecrets();
+                builder..SetProjectPath(env.ContentRootPath).AddUserSecrets();
```

Comment:
Will fix before merging.

- Phil


On Apr 7, 2016, at 5:33 PM, Smit Patel <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/Rules/StarterWeb/AI/IndividualAuth/Startup.cs<https://github.com/aspnet/Templates/pull/485#discussion_r58966757>:

> @@ -34,7 +34,7 @@ public Startup(IHostingEnvironment env)
>              if (env.IsDevelopment())
>              {
>                  // For more details on using the user secret store see http://go.microsoft.com/fwlink/?LinkID=532709
> -                builder.AddUserSecrets();
> +                builder..SetProjectPath(env.ContentRootPath).AddUserSecrets();


is it really ..?

-
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/aspnet/Templates/pull/485/files/0d12ae3d29af13f93a20b332425850dde77740d3#r58966757>